spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -144,4 +144,64 @@
       end
     end
   end
+
+  describe 'GET#show' do
+    context 'when CSV file is valid' do
+      it 'returns success status' do
+        keyword = Fabricate(:keyword_parsed)
+        user = keyword.user
+        get api_v1_keyword_path(keyword.id), headers: create_token_header(user)
```

Comment:
```suggestion

        get api_v1_keyword_path(keyword.id), headers: create_token_header(user)
```

Arrange

Act

Assert

🚀